plugins/system/shortcut/shortcut.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @package     Joomla.Plugins
+ * @subpackage  System.shortcut
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Application\AdministratorApplication;
+use Joomla\CMS\Event\GenericEvent;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\Event\Event;
+use Joomla\Event\SubscriberInterface;
+
+/**
+ * Shortcut plugin to add accessible keyboard shortcuts to the administrator templates.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgSystemShortcut extends CMSPlugin implements SubscriberInterface
+{
+	/**
+	 * Application object.
+	 *
+	 * @var    AdministratorApplication
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected $app;
+
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected $autoloadLanguage = true;
+
+	/**
+	 * Returns an array of events this subscriber will listen to.
+	 *
+	 * The array keys are event names and the value can be:
+	 *
+	 *  - The method name to call (priority defaults to 0)
+	 *  - An array composed of the method name to call and the priority
+	 *
+	 * For instance:
+	 *
+	 *  * array('eventName' => 'methodName')
+	 *  * array('eventName' => array('methodName', $priority))
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public static function getSubscribedEvents(): array
+	{
+		return [
+			'onBeforeCompileHead' => 'initialize',
+			'onLoadShortcuts' => 'addShortcuts'
+		];
+	}
+
+
+	/**
+	 * Add the javascript for the shortcuts
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function initialize()
+	{
+		if (!$this->app->isClient('administrator'))
+		{
+			return;
+		}
+
+		PluginHelper::importPlugin('shortcut');
+
+		$context = $this->app->input->get('option') . '.' . $this->app->input->get('view');
+
+		$shortcuts = [];
+
+		$event = new GenericEvent('onLoadShortcuts', [
+			'context' => $context,
+			'shortcuts' => $shortcuts
+		]);
```

Comment:
```suggestion
		$event = new GenericEvent('onLoadShortcuts', [
				'context'   => $context,
				'shortcuts' => $shortcuts,
			]
		);
```
https://ci.joomla.org/joomla/joomla-cms/55527/1/6